templates/coredns/0/rancher-compose.yml

```diff
@@ -0,0 +1,84 @@
+.catalog:
+  name: "CoreDNS"
+  version: "0.8.0"
```

Comment:
It would be nice if you use the coredns version.
For users, they may not know the meaning of the 0.8 version.

Reply:
Changed to 1.0.1 compatible with coredns version.